core/src/main/java/io/questdb/cairo/wal/WalEventWriter.java

```diff
@@ -299,10 +299,12 @@ int appendSql(int cmdType, CharSequence sqlText, SqlExecutionContext sqlExecutio
         return txn++;
     }
 
-    int invalidate(boolean invalid, @Nullable CharSequence invalidationReason) {
+    int invalidate(long lastRefreshBaseTxn, long lastRefreshTimestamp, boolean invalid, @Nullable CharSequence invalidationReason) {
```

Comment:
nit: should be named `appendMatViewInvalidate`

Reply:
renamed: https://github.com/questdb/questdb/pull/5519/commits/3b0722404833a5b508a78aebc477d80ddd3ccc5e